ACHNBrowserUI/ACHNBrowserUI/views/settings/SettingsView.swift

```diff
@@ -151,6 +159,46 @@ struct SettingsView: View {
         }
     }
     
+    private var notificationSection: some View {
+        Section(header: SectionHeaderView(text: "Notifications", icon: "clock")) {
+            HStack {
+                Toggle(isOn: $appUserDefaults.isShopOpenClose ) {
+                    Text("When shops open/close")
+                }
+                .onTapGesture {
+                    if !self.appUserDefaults.isShopOpenClose {
```

Comment:
Don’t register notifications in settings view. This belongs to `NotificationsManager` in the Backend package

Reply:
Do you know how I should call NotificationsManager? I cannot use an EnvironmentObject